github/libp2p/team_membership.json

```diff
@@ -338,128 +174,23 @@
   "contributors": {
     "BigLep": {
       "role": "member"
-    },
-    "JGAntunes": {
-      "role": "member"
-    },
-    "Kubuxu": {
-      "role": "member"
-    },
-    "MarcoPolo": {
-      "role": "member"
```

Comment:
What's this section for? Should I be readded?

Reply:
This is `team_membership` in `contributors` team.